packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -575,9 +576,13 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
             // DeltaManager overall policy - drop all ops on disconnection and rely on
             // container runtime to deal with resubmitting any ops that did not make it through.
             // So drop them, but also raise error event to look into details.
-            this.logger.sendErrorEvent({
+            this.logger.sendTelemetryEvent({
```

Comment:
I do not think there is a value in this event at all. I.e. it's either an error (i.e. we believe it should not happen), or it's normal flow that is handled, covered by UT, but really does not matter in terms of logging.
Also reason is not package data - it comes from socket (in majority cases) and we freely log it already.